Update plugins install docs for v1.11.0 (#12995)
hashicorp/packer: website/content/docs/plugins/index.mdx

---
description: |
  This topic provides overview information about installing and loading Packer plugins
page_title: Plugins
---

# Plugin installation overview

This topic provides overview information about installing and loading Packer plugins. Plugins are standalone applications that perform additional tasks during each build.

## Introduction

> The Packer binary includes a set of built-in plugins that run automatically, but you can also define a list of external plugins in your template for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code. Packer launches one plugin process for each component in the build.

Reviewer: Just to avoid confusion here, we don't embed plugins with Packer anymore, but there are components that are servable by Packer

```suggestion
The Packer binary includes a set of built-in components that are automatically usable. You can also install external plugins for Packer to run and communicate with throughout the build. These external plugins extend Packer functionality without modifying the core source code. Packer launches one plugin process for each component in the build.
```
